copyright.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -x
```

Comment:
lets put this file in the `etc/` folder so we don't clutter the repo root too much

Reply:
Good point.